internal/planner/planner.go

```diff
@@ -627,6 +567,26 @@ func (p *Planner) RunSubscriptionRequest(
 // resources of the returned plan.
 //
 // @TODO {defradb/issues/368}: Test this exported function.
-func (p *Planner) MakePlan(request *request.Request) (planNode, error) {
-	return p.makePlan(request)
+func (p *Planner) MakePlan(req *request.Request) (planNode, error) {
+	var operation *request.OperationDefinition
+	if len(req.Mutations) > 0 {
+		operation = req.Mutations[0] // @todo: handle multiple mutation operation statements
```

Comment:
todo: is this todo going to be resolved within this PR?

Otherwise please create a ticket and link it here.

Reply:
I copied these over from another function. I'll double check if there is an existing issue or create one if needed.

Reply:
Found it: https://github.com/sourcenetwork/defradb/issues/1395